tsdb/db.go

```diff
@@ -1720,15 +1725,18 @@ func isBlockDir(fi os.FileInfo) bool {
 	return err == nil
 }
 
-// isTmpBlockDir returns dir that consists of block dir ULID and tmp extension.
-func isTmpBlockDir(fi os.FileInfo) bool {
+// isTmpDir returns dir that consists of block dir ULID and tmp extension.
```

Comment:
```suggestion
// isTmpDir returns true if the given file-info contains a block ULID or checkpoint prefix and a tmp extension.
```

The function no longer turns a dir, rather a bool. It also considers the checkpoint prefix, not just block dir ULID.